### DISCUSSION TITLE
Any chance for windows arm64 version?

### DISCUSSION THREAD

**Discussion by hellvy-github** [d2phap/ImageGlass#1377] [General]
Hello, I'm enjoy using ImageGlass on my Desktop PC., thanks for good image viewing app :) 

I'd like to have native arm64 version for my Surface Pro X. Is there any chance for that? Or could you provide some guide for building one from source myself?

Thank you.

**Comment by d2phap** [comment]
I have never tried to build for ARM, but if you wanna try, you need to install a ARM version of Magick.NET (and other 3rd-party libs)

**Reply by hellvy-github** [reply]
Thank you, I'll check it out :)
